<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>center5</class>
 <widget class="QWidget" name="center5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>50</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>行動是治癒恐懼的良藥，而猶豫、拖延將不斷滋養恐懼</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#center {
	background-color: transparent;
    border-image: url(&quot;:/icons/archlinux-light.png&quot;);
}
QWidget:hover#center {
	background-color: transparent;
    border-image: url(&quot;:/icons/archlinux-dark.png&quot;);
}</string>
  </property>
  <widget class="QLabel" name="memUse">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>4</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;华文琥珀&quot;;
color: rgb(30, 30, 30);</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="netUp">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>15</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;Microsoft YaHei&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>0 B</string>
   </property>
  </widget>
  <widget class="QLabel" name="netDown">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>15</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;Microsoft YaHei&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>0 B</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>17</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;华文细黑&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(95, 141, 211);
border-radius: 21px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>11</y>
     <width>106</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 204, 255);
border-top-right-radius:    14px;
border-bottom-right-radius: 14px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/archlinux.png)</string>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>widget</zorder>
  <zorder>memUse</zorder>
  <zorder>netUp</zorder>
  <zorder>netDown</zorder>
  <zorder>percent</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
